.github/workflows/integration.yml

```diff
@@ -274,6 +274,89 @@ jobs:
         with:
           name: wps-jobs-coverage-report
           path: ./wps_jobs/coverage-reports/wps-jobs-coverage-report.xml
+  test-wps-shared:
```

Comment:
Probably something we can do in a follow up ticket but we could make this a reuse-able action I think